.goreleaser.yml

```diff
@@ -16,24 +23,29 @@ builds:
     goarch:
       - 386
       - amd64
-      - arm
       - arm64
     ignore:
       - goos: darwin
         goarch: 386
+      - goos: linux
+        goarch: 386
 archives:
   -
-    name_template: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+    name_template: '{{ .ProjectName }}-{{ .Version }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
     replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
+      darwin: macos
+      386: x86
+      amd64: x64
+    files:
+      - none*
```

Comment:
Whats the purpose of this?

Reply:
By default, this tool adds README and License file. This is discussed [here](https://github.com/goreleaser/goreleaser/issues/602) and added hack for now